index.js

```diff
@@ -218,7 +218,7 @@ exports.writeMainView = function (dir, cb) {
 exports.writeIcon = function (dir, cb) {
   var iconPath = path.join(__dirname, 'assets/icon.png')
```

Comment:
should this be `path.join(__dirname, 'assets', 'icon.png')` since `/` isn't the path separator in windows?  (Does node do the "right thing here" since I jut realized all the require statements use `/` as it is..)